cli/cmd/tables.go

```diff
@@ -63,9 +63,10 @@ func tables(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return fmt.Errorf("failed to get auth token: %w", err)
 	}
+	//TODO: add team name
```

Comment:
Any blockers to adding the team name here as well?

Reply:
Nope, added now - missed it on first pass.